### DISCUSSION TITLE
Backend Theme with BS 5.3 (the color modes)

### DISCUSSION THREAD

**Discussion by patkon** [twbs/bootstrap#38802] [Show & Tell]
Hi Bootstrappers! I've been experimenting a bit with the new color modes and upgraded a backend to Bootstrap 5.3.
What do you guys think?
I am of course grateful for any suggestions. And if you like it, I would be very happy about a star -> https://github.com/SwiftyEdit/SwiftyEdit

Here are some Screenshots

![se_shots-settings](https://github.com/twbs/bootstrap/assets/5982156/4bb5c06a-9f9f-491a-b45e-6edaab69a3b0)
![se_shots-settings-light](https://github.com/twbs/bootstrap/assets/5982156/c8169b7e-9a2f-4eb6-b74e-5adba029358f)
![se_shots_user](https://github.com/twbs/bootstrap/assets/5982156/ae605d87-68dc-4e56-bc25-8907a70956cc)
![se_shots_uploads](https://github.com/twbs/bootstrap/assets/5982156/1fe931aa-a25d-458a-a7ec-b4bb32dc8568)
![se_shot_index](https://github.com/twbs/bootstrap/assets/5982156/63e6c2c7-d781-4148-b38b-e4bd168ce7cb)
![se_shot-dashboard](https://github.com/twbs/bootstrap/assets/5982156/7f107c7c-34d2-4e84-b646-dc6830da2ec6)
![se_shot-edit-page](https://github.com/twbs/bootstrap/assets/5982156/6683401e-13f9-438b-8842-239ea79590bb)

**Comment by julien-deramond** [comment]
Hey @patkon 

Glad to see some color modes in action! Sounds neat!

Don't hesitate if you saw some things we can improve regarding these new color modes when applying them to your project. We already spotted ways to improve them (mainly because we try to avoid too many breaking changes before the v6), but it's always nice to have some feedback from actual projects.

I'm not a designer but here is some really quick personal feedback only based on the provided screenshots.
* You might have some missing gap between the profile and the "?" button, and the theme switcher's height sounds bigger compared to the other elements in the same place
![Screenshot 2023-06-20 at 20 07 43](https://github.com/twbs/bootstrap/assets/17381666/e8f7515b-0947-4549-84cb-d4ee526aa0f4)
* It's maybe because of the low quality of the image but it seems that the red color used for the dark mode is a bit too dark and not contrasted enough. Could be interesting to test a lighter red. Again, maybe just the screenshot gives this impression:
![Screenshot 2023-06-20 at 20 09 21](https://github.com/twbs/bootstrap/assets/17381666/2c4588b5-29d2-4dbe-9c97-55c4d1b24e6c)
![Screenshot 2023-06-20 at 20 11 31](https://github.com/twbs/bootstrap/assets/17381666/d615dcb4-bf8c-4f25-8a86-c6c2557958d3)
* Maybe something's missing here that should be lighter:
![Screenshot 2023-06-20 at 20 12 24](https://github.com/twbs/bootstrap/assets/17381666/ad712c51-570b-40b8-b795-992d95bb63a2)

Keep up the good work.
Dropped a star for the motivation :)

**Reply by patkon** [reply]
**Thank you for your feedback! And of course for the star :-)**
Your suggestions are completely correct, I will improve it. Sometimes it's hard for me to position things correctly (I learned to code all by myself and never had any real training in this area).

I'm already curious how it will be when I have dealt more with the color modes.
Again, thanks for this great library!
